packages/homepage/src/screens/home/hero/Content.js

```diff
@@ -11,93 +10,65 @@ import angular from '../../../assets/icons/home-angular.svg';
-          <img src={vue} alt="Vue Template" />
-        </Sandbox>
-        <Sandbox
-          href="/s/angular"
-          title="Angular"
-          style={{
-            animationDelay: '0.8s',
-          }}
-        >
-          <img src={angular} alt="angular Template" />
-        </Sandbox>
-        <Sandbox
-          href="/s/github/codesandbox-app/static-template/tree/master/"
-          title="HTML 5"
-          style={{
-            animationDelay: '0.9s',
-          }}
-        >
-          <img src={html} alt="HTML Template" />
-        </Sandbox>
-        <Sandbox href="/s" title="More" style={{ animationDelay: '1.0s' }}>
-          <img src={more} alt="More Template" />
-        </Sandbox>
-      </SandboxButtons>
+const Content = () => (
+  <>
+    <SandboxButtons>
+      <Sandbox href="/s/new" title="React" style={{ animationDelay: '0.5s' }}>
+        <img src={react} alt="React Template" />
+      </Sandbox>
+      <Sandbox
+        href="/s/vanilla"
+        title="Vanilla"
+        style={{
+          animationDelay: '0.6s',
+        }}
+      >
+        <img src={vanilla} alt="Vanilla Template" />
+      </Sandbox>
+      <Sandbox href="/s/vue" title="Vue" style={{ animationDelay: '0.7s' }}>
+        <img src={vue} alt="Vue Template" />
+      </Sandbox>
+      <Sandbox
+        href="/s/angular"
+        title="Angular"
+        style={{
+          animationDelay: '0.8s',
+        }}
+      >
+        <img src={angular} alt="angular Template" />
+      </Sandbox>
+      <Sandbox
+        href="/s/github/codesandbox-app/static-template/tree/master/"
+        title="HTML 5"
+        style={{
+          animationDelay: '0.9s',
+        }}
+      >
+        <img src={html} alt="HTML Template" />
+      </Sandbox>
+      <Sandbox href="/s" title="More" style={{ animationDelay: '1.0s' }}>
+        <img src={more} alt="More Template" />
+      </Sandbox>
+    </SandboxButtons>
 
-      <motion.div
-        initial={{ opacity: 0, y: 20 }}
-        animate={{ opacity: 1, y: 0 }}
-        transition={{ duration: 0.5, ease: 'easeOut', delay: 0.5 }}
+    <motion.div
+      initial={{ opacity: 0, y: 20 }}
+      animate={{ opacity: 1, y: 0 }}
+      transition={{ duration: 0.5, ease: 'easeOut', delay: 0.5 }}
+    >
+      <Button
+        style={{
+          padding: '.75rem 2rem',
+          marginBottom: '.5rem',
+          borderRadius: '.25rem',
+        }}
+        href="/s"
       >
-        <Button
-          style={{
-            padding: '.75rem 2rem',
-            marginBottom: '.5rem',
-            borderRadius: '.25rem',
-          }}
-          href="/s"
-        >
-          {freeWordingA ? (
-            'Create Sandbox, it’s free'
-          ) : (
-            <>
-              <strong>{'</>'}</strong> Start coding for free
-            </>
-          )}
-        </Button>
-      </motion.div>
-    </>
-  );
-};
+        <strong>{'</>'}</strong> Start coding for free
```

Comment:
@tromika is this the right one right?

Reply:
@danilowoz yep, that's the right one! Thanks!